packages/core/src/App/app.js

```diff
@@ -12,7 +12,7 @@ const setStorageEvents = root_store => {
             case 'client.accounts': {
                 const active_loginid = root_store.client.loginid;
                 const new_currency = JSON.parse(evt.newValue)[active_loginid]?.currency;
-                const old_currency = JSON.parse(evt.oldValue)[active_loginid]?.currency;
+                const old_currency = JSON.parse(evt.oldValue)?.[active_loginid]?.currency;
```

Comment:
It seems it is a good idea to add the same optional-chaining operator into the upper line (14) for `new_currency` too to prevent a future bug.